### PR TITLE
Feat/slack bot

### PR DESCRIPTION
**Pull request by jvargas98**

### CODE REVIEW

File: helpers/passwords.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const generateRandom = () => Math.random().toString(36).slice(-8);
```
Comment (LoginALee) [inline helpers/passwords.js:1]: Cool detail!

**Review by LoginALee — APPROVED** [review]
Great job Jorge!! :rocket:  :robot:

File: routes/slack.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
+const express = require('express');
```
Comment (OmauryMarmolejo) [inline routes/slack.js:1]: Maybe we can move a good part of this code to a controller/service

File: config/payloads.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+const modal = {
```
Comment (ese-varo) [inline config/payloads.js:1]: Nice!, putting the modals on its own files :100:

**Review by ese-varo — APPROVED** [review]
Nice job! :rocket:

**Review by ese-varo — APPROVED** [review]
:rocket:

**Review by GregVaz — APPROVED** [review]
Nice work :rocket:

**Pull request merged**